server/notification-providers/cellsynt.js

```diff
@@ -26,11 +23,13 @@ class Cellsynt extends NotificationProvider {
         };
         try {
             const resp = await axios.post("https://se-1.cellsynt.net/sms.php", null, data);
-            if (resp.data == null || resp.data.includes("Error")) {
+            if (resp.data == null ) {
+                throw new Error("Error: Could not connect to Cellsynt, please try again.");
```

Comment:
`Error:` is already appended via the outer try-catch

```suggestion
                throw new Error("Could not connect to Cellsynt, please try again.");
```